tests/foreman/api/test_computeresource_gce.py

```diff
@@ -65,6 +65,26 @@ def module_gce_finishimg(
     return finish_image
 
 
+@pytest.fixture(scope='module')
+def module_gce_finishimg_puppet(
```

Comment:
for a second I thought it should be `finishing`, maybe `finish_img` would read nicer?

Reply:
Thanks for your input @pondrejk I will keep this in mind for upcoming PR's and replicate universally.